config/config.go

```diff
@@ -0,0 +1,25 @@
+package config
+
+import "flag"
+
+var (
+	debugServer bool
+	debugPort   int
+)
+
+func init() {
+	flag.BoolVar(&debugServer, "debug-server", false, "enable the debug server")
```

Comment:
should we only use `debug-port` and only enable the debug server if the port is provided ?

Reply:
We wouldn't be able to use a default value for the port then